### DISCUSSION TITLE
ApiKey problem

### DISCUSSION THREAD

**Discussion by Zilbon** [DependencyTrack/dependency-track#1016] [Q&A]
Hello,
Does someone can tell me if there is a problem with my code or if the problem is on the api side. I keep getting the error  "apikey missing", but my apikey is in my code. I am using angular typescript.
Here is the code:
 ngOnInit(): void{ 
  
    
     let headers = {'Content-type':'application/x-www-form-urlencoded; charset=UTF-8'} 
     let body ={'X-VulDB-ApiKey':'****************','advancedsearch': 'cve:CVE-2014-6271',}

    this.http.post('https://vuldb.com/?api', body, {headers} ).subscribe(data =>{
      console.log(data);
    })

}

**Comment by stevespringett** [comment]
I think you may want to report this issue to VulnDB support. The Dependency-Track project contains a client component for VulnDB in the same way that your Angular app is a client.

**Reply by Zilbon** [reply]
I already report it to them but I don't know if the problem come from my side. I have been working on this simple request for 6 days now. I will take a look on that project maybe it will help me. Thank you!
